compute_sdk/tests/unit/test_login_manager.py

```diff
@@ -180,8 +186,6 @@ def test_is_client_login():
             assert not is_client_login()
 
     for cid, csc in (
```

Comment:
This tests both the 'ok' and 'raise error' paths.  Prob better to separate the test methods (and name each more specifically) but as it's pre-existing, ok for now.

Reply:
I have an upcoming PR that may wipe this test out, so I'll leave it as-is.